src/SVGSkin.js

```diff
@@ -147,7 +165,7 @@ class SVGSkin extends Skin {
         // Can't use bitwise stuff here because we need to handle negative exponents
         const mipScale = Math.pow(2, mipLevel - INDEX_OFFSET);
 
-        if (this._svgRenderer.loaded && !this._scaledMIPs[mipLevel]) {
+        if (this._svgImage.complete && !this._scaledMIPs[mipLevel]) {
```

Comment:
It's not clear to me that this is guaranteed to never become true before the `load` event handler is called. I think to be safe we should set a flag in the `load` event handler and check that here, similar to how the `loaded` flag worked before.

Reply:
I've done so now. This required some additional supporting changes to avoid a race condition in case `setSVG` is called a second time before the original image is finished loading.